spec/index.emu

```diff
@@ -87,13 +92,9 @@ location: https://rawgit.com/tc39/proposal-realms/master/index.html
           1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
           1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
           1. Let _callerContext_ be the running execution context.
-          1. Let _newContext_ be a new execution context.
-          1. Set the Function of _newContext_ to *null*.
-          1. Set the Realm of _newContext_ to _O_.[[Realm]].
-          1. Set the ScriptOrModule of _newContext_ to *null*.
-          1. Push _newContext_ onto the execution context stack; _newContext_ is now the running execution context.
+          1. Push _O_.[[ExecutionContext]] onto the execution context stack; _O_.[[ExecutionContext]] is now the running execution context.
```

Comment:
What are the implications of sharing the execution context per import call? I believe none, just want to confirm!

Reply:
There are none; the execution context is never mutated. It's just an editorial convenience. But, note, this context is also used by the HTML integration. I could have the HTML integration make its own context instead, if you all prefer; these are minor editorial differences.